### DISCUSSION TITLE
Congrats to v.1.0.0 FINAL :) !!

### DISCUSSION THREAD

**Discussion by LeeBinder** [mucommander/mucommander#611] [General]
![fw](https://user-images.githubusercontent.com/39203497/163887780-f58a807c-7c0c-4df2-a4a7-379d9445be2b.gif)

🥇

**Comment by mj162** [comment]
Yes, thank you!  Have been using through:
```
drwxr-xr-x 5  16384 Jan  2  2021 mucommander-0.9.6-1
drwxr-xr-x 6  16384 Jul  7  2022 mucommander-0.9.7-1
drwxr-xr-x 6  16384 Jul  4  2022 mucommander-1.0.0-1
drwxr-xr-x 6  16384 Dec  6  2022 mucommander-1.1.0-1
drwxr-xr-x 6  16384 Jun 29 13:53 mucommander-1.2.0-1
```
on Solaris SPARC and Linux (never got round to installing on openvms).  Yes, a few issues with setting up action commands to work as I want, but the cross-platform work-across-updates is really appreciated.  In the time, other apps have broken and haven't had the time to help back on feet, but muCommander keeps going.  Thanks.

**Reply by ahadas** [reply]
it's great to read this, thanks for the feedback and for sharing that @mj162 !

**Comment by mj162** [comment]
No, thank you for investing your skill and time to the benefit of the wider community.
